src/gflownet/trainer.py

```diff
@@ -119,13 +121,12 @@ def __init__(self, hps: Dict[str, Any], device: torch.device):
         # The final config is obtained by merging the three sources
         self.cfg: Config = OmegaConf.structured(Config())
         self.set_default_hps(self.cfg)
-        OmegaConf.merge(self.cfg, hps)
+        # OmegaConf returns a fancy object but we can still pretend it's a Config instance
+        self.cfg = OmegaConf.merge(self.cfg, hps)  # type: ignore
 
         self.device = device
-        # idem, but from `self.test_data` during validation.
-        self.valid_offline_ratio = 1
         # Print the loss every `self.print_every` iterations
-        self.print_every = 1000
```

Comment:
Was this set for debugging, then you forgot, or the desired default every step is now `1`

Reply:
Yes, actually I'll make that a setting for future debugging.